scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -392,6 +392,63 @@ VQFN-24-1EP_4x4mm_P0.5mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+VQFN-24-1EP_4x4mm_P0.5mm_EP2.5x2.5mm:
+  device_type: 'VQFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.nxp.com/docs/en/package-information/SOT616-3.pdf'
+  ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    minimum: 3.9
+    nominal: 4.0
+    maximum: 4.1
+  body_size_y:
+    minimum: 3.9
+    nominal: 4.0
+    maximum: 4.1
+  overall_height:
+    maximum: 1
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 2.45
+    nominal: 2.5
+    maximum: 2.75
+  EP_size_y:
+    minimum: 2.45
+    nominal: 2.5
+    maximum: 2.75
```

Comment:
Please use package pad dimensions as given in the datasheet and use EP_size_[xy]_overwrite to get a different FP pad size if necessary.

Reply:
I don't fully understand your request. EP size is already specified as given in the package datasheet (min 2.45mm, max 2.75mm), nominal size was removed (as not given in the datasheet) but it was derived from the min and max values and conformed with the nominal pad size (2.5x2.5mm) given in the datasheet (fig. 2, page 3).